app.js

```diff
@@ -1,15 +1,28 @@
 const { getPool } = require('./db');
 const http = require('http');
+const { getAllCostumes } = require('./db/costumes');
 
 const pool = getPool();
 
 const PORT = process.env.PORT || 5000;
 const HOST = process.env.HOST || '127.0.0.1';
 
 const server = http.createServer( async (req, res) => {
-    res.writeHead(200, {'Content-Type': 'text/plain'});
-    res.write('Server is running');
-    res.end();
+   if (req.url.startsWith('/api')) {
+    if(req.url === '/api/costumes' && req.method === 'GET') {
+        try {
+            const costumes = await getAllCostumes(pool);
```

Comment:
I thought we weren't going to pass `pool` to the DB adapters in costume-shop? Heads up that this architecture change snuck past me because part 2 PR was quite big. Worth filing a new issue to do a refactor at a later point. (not blocking for this PR)

Reply:
Apologies. Remind me, did you want us to use `client` instead or simply want it structured so that neither `pool` nor `client` needs to be passed to the adapters?

Reply:
Pool is fine. The idea was to not pass pool as an argument but instead have it returned in its module exports, and then any file that used pool imports pool.js

Fwiw I think this would be easier if we moved the create tables function out of index and into its own init_db.js file

This isn't showstopper stuff, just means we don't have to pass pool all over, which makes the code more DRY